### PR TITLE
Add new nightly job

### PR DESCRIPTION
**Pull request by pcreech**
Add new nightly job that utilizes the new ansible based pulp-packaging repo

### CODE REVIEW

File: ci/jjb/jobs/pulp-packaging.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# This set of Jenkins jobs creates an installs all the
```
Comment (elyezer) [inline ci/jjb/jobs/pulp-packaging.yaml]: `creates and installs`? (see the `d`)

File: ci/jjb/jobs/pulp-packaging.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+- job-template:
+    name: 'pulp-packaging-{release_config}'
+    defaults: ci-workflow-runtest
+    node: 'rhel7-np'
+    properties:
+        - build-ownership
+    scm:
+        - git:
+            url: 'https://github.com/pcreech/pulp-packaging.git'
```
Comment (elyezer) [inline ci/jjb/jobs/pulp-packaging.yaml]: Is yours pulp packaging different from this repo?

Comment (pcreech) [reply ci/jjb/jobs/pulp-packaging.yaml]: Yes!  I am working on a new pulp-packaging repo that will supercede the python scripts we use now, and will mirror similar efforts by foreman and katello teams.  It isn't 'official' yet, and after a bit of final polish, I will be presenting this to pulp to get it added to pulp.

Comment (elyezer) [reply ci/jjb/jobs/pulp-packaging.yaml]: Great thank you

**Pull request merged**